spec.md

```diff
@@ -1847,6 +1847,33 @@ To disallow infinitely recursive hardware, modules cannot contain instances of
 itself, either directly, or indirectly through instances of other modules it
 instantiates.
 
+## Remote Instances
+
+FIRRTL modules may also be remotely instantiated.  A remotely instantiated
+module will cause the instantiation to be emitted outside its instantiating
+module.  (See the FIRRTL ABI for more information on this lowering.)
+
+Remotely instantiated modules must only contain input ports of passive types or
```

Comment:
eats shoots and leaves? Would a comma before the or be more clarifying? I'm not sure if I am allowed to have an input probe port...?